webextensions/api/devtools.json

```
@@ -31,7 +31,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/devtools/inspectedWindow",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "19"
```

:x: https://github.com/chromium/chromium/commit/2cf725f68fa5106fd5bcb1e1fb8699bb3628cafe first appeared in [18.0.1015.0](https://github.com/chromium/chromium/releases/tag/18.0.1015.0) before 18[.0.1025].

```suggestion
                "version_added": "18"
```